packages/react-scripts/scripts/utils/verifyTypeScriptSetup.js

```diff
@@ -172,14 +172,17 @@ function verifyTypeScriptSetup() {
 
     parsedCompilerOptions = result.options;
   } catch (e) {
-    console.error(
-      chalk.red.bold(
-        'Could not parse',
-        chalk.cyan('tsconfig.json') + '.',
-        'Please make sure it contains syntactically correct JSON.'
-      )
-    );
-    console.error(e && e.message ? `Details: ${e.message}` : '');
+    if (e && e.name === 'SyntaxError') {
+      console.error(
+        chalk.red.bold(
+          'Could not parse',
+          chalk.cyan('tsconfig.json') + '.',
+          'Please make sure it contains syntactically correct JSON.'
+        )
+      );
+    }
+    
+    console.error(e && e.message ? `${e.message}` : '');
```

Comment:
Do we want to `console.error()` again even if it's a `SyntaxError`?

Reply:
Good point. `console.log` is probably fine here.